_historical_review/core-tasks/app/services/neo4j_graph_queries.py

```
@@ -0,0 +1,561 @@
+            RETURN DISTINCT neighbor.entity_id as neighbor_id,
+                   neighbor.doc_id as neighbor_id,
+                   labels(neighbor)[0] as neighbor_type,
+                   type(r) as relationship,
+                   CASE
+                       WHEN startNode(r) = e THEN 'outgoing'
+                       ELSE 'incoming'
+                   END as direction
+            """
+
+            results = self.connection.execute_query(query, {"entity_id": entity_id})
+            return results
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)